File: LeetCode_Solutions/2727-number-of-senior-citizens/number-of-senior-citizens.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        cnt=0
        for i in range(len(details)):
            if details[i][-4]>='6':
                if details[i][-4]=='6' and details[i][-3]>'0':
                    cnt+=1
                elif details[i][-4]>'6':
                    cnt+=1
        return cnt

File: LeetCode_Solutions/2727-number-of-senior-citizens/number-of-senior-citizens.java
class Solution {
    public int countSeniors(String[] details) {
        int cnt=0;
        for(int i=0;i<details.length;i++){
            int ind1=details[i].length()-4;
            int ind2=details[i].length()-3;
            if(details[i].charAt(ind1)>='6'){
                if(details[i].charAt(ind1)=='6'&&details[i].charAt(ind2)>'0'){
                     cnt++;
                }
               else if(details[i].charAt(ind1)>'6'&&details[i].charAt(ind2)>='0'){
                    cnt++;
                }
            }
        }
        return cnt;
    }
}